openapi/covid19/Module.md

```diff
@@ -1,9 +1,9 @@
 ## Overview
 
-Ballerina connector for Covid-19 consumes the data exposed in https://disease.sh. It provides easy access to latest Covid-19 related data across the world
- 
-This module supports Novel Covid-19 API version 3.0.0. 
- 
+This is a generated connector from [Novel Covid19 API version 3.0.0](https://disease.sh/docs/) OpenAPI Specification.
```

Comment:
Please update all doc comments to `COVID-19` appropriately. Refer https://github.com/wso2-enterprise/choreo/issues/6490
```suggestion
This is a generated connector from [Novel COVID-19 API version 3.0.0](https://disease.sh/docs/) OpenAPI Specification.
```